### ISSUE TITLE
`getindex(::LaTeXStrings.LaTeXString, ::AbstractVector{Bool})` is ambiguous (in Julia v1.7)

### ISSUE THREAD

**Issue by ranocha** [stevengj/LaTeXStrings.jl#61]
I found this while running tests with [Aqua.jl](https://github.com/JuliaTesting/Aqua.jl) for one of my packages.
```julia
julia> using Pkg; Pkg.activate(temp=true); Pkg.add("LaTeXStrings")
  Activating new project at `/tmp/jl_PC9tEj`
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
    Updating `/tmp/jl_PC9tEj/Project.toml`
  [b964fa9f] + LaTeXStrings v1.3.0
    Updating `/tmp/jl_PC9tEj/Manifest.toml`
  [b964fa9f] + LaTeXStrings v1.3.0

julia> using LaTeXStrings

julia> getindex(L"\alpha \beta \gamma", [true])
ERROR: MethodError: getindex(::LaTeXString, ::Vector{Bool}) is ambiguous. Candidates:
  getindex(s::AbstractString, v::AbstractVector{Bool}) in Base at strings/basic.jl:194
  getindex(s::LaTeXString, i::AbstractVector{<:Integer}) in LaTeXStrings at ~/.julia/packages/LaTeXStrings/pJ7vn/src/LaTeXStrings.jl:120
Possible fix, define
  getindex(::LaTeXString, ::AbstractVector{Bool})
Stacktrace:
 [1] top-level scope
   @ REPL[8]:1
```

**Comment by stevengj** [comment]
Note that this method [just throws an error for strings](https://github.com/JuliaLang/julia/blob/ae89f8b00a6e785832e4f47de73599e032aa9674/base/strings/basic.jl#L194-L195) anyway … what are you trying to accomplish here?

**Comment by ranocha** [comment]
I ran the `Auqa.test_all` tests from [Aqua.jl](https://github.com/JuliaTesting/Aqua.jl) for one of my packages. It reported this as ambiguous method. Thus, I needed to exclude `getindex` explicitly from the method ambiguity tests which is a bit annoying since it excludes checks of the same function for my own types, too. Thus, it may be nice to resolve this method ambiguity to allow downstream packages to run all ambiguity tests with Aqua.jl. 
If this would be okay for you, I can make a PR. Otherwise, feel free to close this issue.
